monoio/src/driver/op/read.rs

```diff
@@ -188,29 +220,7 @@ impl<T: IoVecBufMut> OpAble for ReadVec<T> {
 
     #[cfg(all(any(feature = "legacy", feature = "poll-io"), windows))]
     fn legacy_call(&mut self) -> io::Result<u32> {
-        let mut nread = 0;
-        let mut flags = 0;
-        let ret = unsafe {
-            WSARecv(
-                self.fd.raw_socket() as _,
-                self.buf_vec.write_wsabuf_ptr(),
-                self.buf_vec.write_wsabuf_len().min(u32::MAX as usize) as _,
-                &mut nread,
-                &mut flags,
-                std::ptr::null_mut(),
-                None,
-            )
-        };
-        match ret {
-            0 => Ok(nread),
-            _ => {
-                let error = unsafe { WSAGetLastError() };
-                if error == WSAESHUTDOWN {
-                    Ok(0)
-                } else {
-                    Err(io::Error::from_raw_os_error(error))
-                }
-            }
-        }
+        // There is no `readv` like syscall of file on windows.
+        unimplemented!()
```

Comment:
Can it fallback to read?

Reply:
on windows, I use [`read` to simulate the `readv`](https://github.com/Lzzzzzt/monoio/blob/99a3c23ae750a1dc4b3cdaaf26bf9ff2b124a304/monoio/src/fs/file/windows.rs#L30-L73), this function will not be called, so I think there is no need to fallback to read.
BTW, it's also ok to mv [this](https://github.com/Lzzzzzt/monoio/blob/99a3c23ae750a1dc4b3cdaaf26bf9ff2b124a304/monoio/src/fs/file/windows.rs#L30-L73) to the implement of `ReadVec`

Reply:
If readv is not used on windows, how about add condition compile flags for ReadVec/WriteVec?

Reply:
sounds good. 
And can we separate it into another file?

Reply:
Sry for my mistake, on windows, the `WriteVec/ReadVec` are used to read/write `TcpStream`.
But IMO, it's a little bit wired.